src/haz3lweb/view/dhcode/layout/DHDoc_Exp.re

```diff
@@ -382,7 +382,10 @@ let rec mk =
             DHDoc_common.Delim.close_Fun,
           ]);
         } else {
-          annot(DHAnnot.Collapsed, text("<fn>"));
+          switch (s) {
+          | None => annot(DHAnnot.Collapsed, text("<fn>"))
```

Comment:
this is ambiguous if we have a function named `fn`, let's use `<anon fn>` instead

Reply:
Sure. I have changed to use <anon fn> instead.